### ISSUE TITLE
Unable to properly clear multiple custom_func filter types?

### ISSUE THREAD

**Issue by xmikew** [vedmack/yadcf#226]
Hello,

See the following fiddle: http://jsfiddle.net/k696o3j0/

So the issue is that the custom_func pushes onto the native DataTables filtering API which is global?
This means that if I have two custom_func filters defined, they both get called when I change make a change to the filter drop down. This is because regardless, the clear option is set to -1, so there is no way to know which column I have cleared when selecting no option (and having disabled the clear button).

perhaps the default value can be unique per column so we know which one we are clearing? I'll admit the interface on defining custom functions on a DataTable is a little lost on me. I'm trying to filter on booleans but making boolean more friendly, "yes" or "no" vs. true and false.

```
if (columnObj.filter_type === "select" || columnObj.filter_type === 'custom_func' || columnObj.filter_type === "multi_select" || columnObj.filter_type === 'multi_select_custom_func') {
   if (columnObj.filter_type === "select" || columnObj.filter_type === 'custom_func') {
      options_tmp = "<option value=\"" + "-1" + "\">" + filter_default_label + "</option>";
      .
      .
      .
```

Thanks, let me know if there is something I can do to help.

\- Mike

**Comment by vedmack** [comment]
fixed in [0.8.9.beta.12](https://github.com/vedmack/yadcf/blob/1bd84fe4fb32a9ec58ac1b1a68582f950205160a/jquery.dataTables.yadcf.js) see it working http://jsfiddle.net/k696o3j0/27/

**Comment by xmikew** [comment]
Hi, 

Thanks so much for the quick fix and the lib. Much appreciated.

- Mike
